.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "python.linting.enabled": true,
+    "python.testing.pytestArgs": [
+        "tests"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "[python]": {
+        "editor.defaultFormatter": "ms-python.black-formatter",
+        "editor.formatOnSave": true
+    },
+    "python.formatting.provider": "none"
```

Comment:
if we are going to do this, also do trimOnSave

Reply:
The closest I found was trimTrailingWhitespace